cli/flox/src/commands/activate.rs

```diff
@@ -245,6 +235,16 @@ impl Activate {
             ("FLOX_PROMPT_COLOR_2", prompt_color_2),
         ]);
 
+        // Set `FLOX_PROMPT_ENVIRONMENTS` to either the constructed prompt string
+        // or "" if the prompt is disabled.
+        //
+        // The latter is necessary for inner actiavtions to override the policy
+        if let Some(flox_prompt_environments) = flox_prompt_environments {
+            exports.insert(FLOX_PROMPT_ENVIRONMENTS_VAR, flox_prompt_environments);
+        } else {
+            exports.insert(FLOX_PROMPT_ENVIRONMENTS_VAR, "".to_string());
+        }
```

Comment:
nit: leave this inside all the exports above
```
            // Set `FLOX_PROMPT_ENVIRONMENTS` to either the constructed prompt string
            // or "" if the prompt is disabled.
            //
            // The latter is necessary for inner actiavtions to override the policy
            (
                FLOX_PROMPT_ENVIRONMENTS_VAR,
                flox_prompt_environments.unwrap_or("".to_string()),
            ),
```

Reply:
yeah the logic was a bit different before (had a new variable `FLOX_DISABLE_PROMT` that would _also_ be set, i agree with a single one we should probably just move it back